### PR TITLE
[TECH] Admin page - Distribution of tests

### PR DESCRIPTION
**Pull request by snaf-trip**

### CODE REVIEW

**Comment by mtalstykh-exadel** [comment]
check how other PRs naming look like

image

File: src/pages/admin/test-page-admin/test-page-admin.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.paper {
```
Comment (mtalstykh-exadel) [inline src/pages/admin/test-page-admin/test-page-admin.scss]: that style will affect all .paper elements across whole project

File: src/App.js [hunk 2]
```diff
@@ -20,8 +21,9 @@ const App = () => {
         <PrivateRoute path="/buttons"><Buttons /></PrivateRoute>
         <PrivateRoute path="/counter"><Counter /></PrivateRoute>
         <PrivateRoute path="/profile"><Profile /></PrivateRoute>
-        <PrivateRoute path="/" exact><Main /></PrivateRoute>
+        <PrivateRoute path="/main" ><Main /></PrivateRoute>
```
Comment (mtalstykh-exadel) [inline src/App.js]: why you changed it?

File: src/styles/normalize.css [hunk 3]
```diff
@@ -177,7 +177,8 @@ textarea {
  */
 
 button,
-input { /* 1 */
```
Comment (mtalstykh-exadel) [inline src/styles/normalize.css:180]: no need to format it)

File: src/pages/admin/test-page-admin/tests-page-admin.jsx [hunk 4]
```diff
@@ -0,0 +1,179 @@
...
+    paddingTop: "auto",
+    paddingLeft: 21,
+  }
+}))(Select);
+
+const ButtonAssign = withStyles(() => ({
+  root: {
+    minWidth: 127,
+  }
+}))(Button);
```
Comment (mtalstykh-exadel) [inline src/pages/admin/test-page-admin/tests-page-admin.jsx]: use SCSS to style components

File: package.json [hunk 5]
```diff
@@ -5,6 +5,7 @@
   "dependencies": {
     "@lingui/react": "^3.10.2",
     "@material-ui/core": "^4.12.0",
+    "@material-ui/data-grid": "^4.0.0-alpha.33",
```
Comment (mtalstykh-exadel) [inline package.json]: looks like you dont use it

File: src/pages/adminDistribution/tests-page-admin.jsx [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(+event.target.value);
+    setPage(0);
+  };
+
+  const testAction = (action) => {
+    console.log(action);
+  };
```
Comment (tensenrey) [inline src/pages/adminDistribution/tests-page-admin.jsx]: what is it?

File: src/pages/adminDistribution/tests-page-admin.jsx [hunk 7]
```diff
@@ -0,0 +1,148 @@
...
+                                    keysForOptions++;
+                                    return (
+                                      <option key={keysForOptions} value={coachName}>
+                                        {coachName}
+                                      </option>
+                                    );
+                                  })}
+                                </Select>
+                              ) : (
+                                console.log('false')
```
Comment (tensenrey) [inline src/pages/adminDistribution/tests-page-admin.jsx]: what is it?
use return null;

File: src/App.js [hunk 8]
```diff
@@ -7,6 +7,7 @@ import PrivateRoute from './utils/privateRoute';
 import Profile from './pages/profile/profile';
 import Main from './pages/main/main';
 import DemoJWT from './pages/jwt/jwt.jsx';
+import AdminTestPage from './pages/adminDistribution/tests-page-admin.jsx';
```
Comment (mtalstykh-exadel) [inline src/App.js]: why not 
./pages/AdminDistribution/AdminDistribution.jsx

or 

./pages/Tests-page-admin/Tests-page-admin.jsx?

File: src/App.js [hunk 9]
```diff
@@ -21,6 +22,7 @@ const App = () => {
         <PrivateRoute path="/profile"><Profile /></PrivateRoute>
         <PrivateRoute path="/" exact><Main /></PrivateRoute>
         <PrivateRoute path="/jwt" ><DemoJWT /></PrivateRoute>
+        <PrivateRoute path="/admindistribution" ><AdminTestPage /></PrivateRoute>
```
Comment (mtalstykh-exadel) [inline src/App.js]: use "-" to separate words >>> admin-distribution

File: src/pages/adminDistribution/tests-page-admin.jsx [hunk 10]
```diff
@@ -0,0 +1,148 @@
+import React, { useState } from "react";
+import Layout from '../../components/Layout/Layout.js';
+import Paper from "@material-ui/core/Paper";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableContainer from "@material-ui/core/TableContainer";
```
Comment (mtalstykh-exadel) [inline src/pages/adminDistribution/tests-page-admin.jsx]: use such a style of imports

import { AppBar, Toolbar, bla bla bla, IconButton, Avatar, Badge, blablabla, Drawer } from '@material-ui/core';

File: src/pages/adminDistribution/tests-page-admin.jsx [hunk 11]
```diff
@@ -0,0 +1,148 @@
...
+    setPage(newPage);
+  };
+
+  const handleChangeRowsPerPage = (event) => {
+    setRowsPerPage(+event.target.value);
+    setPage(0);
+  };
+
+  const testAction = (action) => {
+    console.log(action);
```
Comment (mtalstykh-exadel) [inline src/pages/adminDistribution/tests-page-admin.jsx]: remove logs

File: src/App.js [hunk 12]
```diff
@@ -8,6 +8,7 @@ import Profile from './pages/profile/profile';
 import Main from './pages/main/main';
 import Page404 from "./pages/Page404/Page404";
 import EditTests from './pages/editTests/editTests';
+import AdminDistribution from './pages/adminDistribution/adminDistribution';
```
Comment (mtalstykh-exadel) [inline src/App.js]: './pages/**A**dminDistribution/**A**dminDistribution';

File: src/pages/adminDistribution/adminDistribution.jsx [hunk 13]
```diff
@@ -0,0 +1,134 @@
+import React, { useState } from "react";
```
Comment (mtalstykh-exadel) [inline src/pages/adminDistribution/adminDistribution.jsx]: pages/AdminDistribution/AdminDistribution.jsx

**Review by tensenrey — DISMISSED** [review]

**Comment by mtalstykh-exadel** [comment]
fix all cooments

File: src/styles/normalize.css (hunk 3 above)
Comment (snaf-trip) [reply src/styles/normalize.css:180]: i fixed it

File: src/pages/AdminDistribution/AdminDistribution.scss [hunk 14]
```diff
@@ -0,0 +1,24 @@
+.AdminDistribution {
+  .paper {
+    margin-top: 40px;
+  }
+
+  .headItems {
+    font-family: "Roboto bold", sans-serif;
```
Comment (mtalstykh-exadel) [inline src/pages/AdminDistribution/AdminDistribution.scss]: use just font-weight: bold;

File: src/testData/Coaches.js [hunk 15]
```diff
@@ -0,0 +1,2 @@
+const coaches = ['Coach1', 'Coach2', 'Coach3', 'Coach4', 'Coach5'];
```
Comment (mtalstykh-exadel) [inline src/testData/Coaches.js]: move to the component folder

**Review by mtalstykh-exadel — APPROVED** [review]

**Review by tensenrey — APPROVED** [review]

**Pull request merged**